cdap-docs/developers-manual/source/getting-started/standalone/docker.rst

```diff
@@ -178,8 +178,42 @@ Docker and CDAP Applications
 
 .. include:: ../dev-env.rst  
    :start-line: 7
+   :end-line: 23
 
-.. include:: ../start-stop-cdap.rst  
-   :start-line: 4
+.. tabbed-parsed-literal::
+   :tabs: "Linux Virtual Machine"
+   :independent:
+
+   $ mvn archetype:generate \
+       -DarchetypeGroupId=co.cask.cdap \
+       -DarchetypeArtifactId=cdap-app-archetype \
+       -DarchetypeVersion=\ |release|
+
```

Comment:
Actually, I think your first question (Why is this here?) is more germane...I've run the Docker on Mac, will inflict it on my wife's Windows machine, and a linux box, and update everything accordingly...I see what you mean about how different all the approaches are, depending on the host OS.